packages/hdwallet-ledger/package.json

```diff
@@ -28,7 +28,7 @@
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@ledgerhq/hw-app-btc": "^6.7.0",
+    "@ledgerhq/hw-app-btc": "^10.0.8",
```

Comment:
BTC calls currently failing with `CLA_NOT_SUPPORTED (0x6e00)` i.e the newer Bitcoin app version on the ledger is incompatible with the older `@ledgerhq/hw-app-btc`, meaning the method is effectively not understood.
This fixes it, confirmed with sandbox:
<img width="422" alt="image" src="https://github.com/shapeshift/hdwallet/assets/17035424/61574779-cd35-499a-808b-694272d2519d">